biblio.json.bak

```diff
@@ -1,4 +1,12 @@
 {  
+  "Ceremony": {
+    "title": "Ceremony Design and Analysis",
+    "href": "https://eprint.iacr.org/2007/399.pdf",
+    "authors": ["Carl Ellison"],
+    "date": "2007"
+  },
+
+	
   "GeoJSON": {
     "title": "The GeoJSON Format Specification",
     "href": "http://geojson.org/geojson-spec.html"
```

Comment:
i"m not sure what you're referring to by "This should be deleted too" ?

WRT the PDF diff file -- yeah, that is an experiment for including a textual diff of the changes in the PR.  if it doesn't work for folks then we can remove it and the /diffs subdir.